lib/WeBWorK/PG/IO.pm

```diff
@@ -131,7 +132,8 @@ sub read_whole_problem_file {
 
 sub read_whole_file {
 	my $filePath = shift;
-
+	warn "<br/>Can't read file $filePath<br/>" unless -r $filePath;
```

Comment:
I was making it stand out during testing. I'll remove it.

Sent from my iPhone

On Oct 15, 2015, at 3:34 PM, Geoff Goehle <notifications@github.com<mailto:notifications@github.com>> wrote:


In lib/WeBWorK/PG/IO.pm<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_openwebwork_pg_pull_245-23discussion-5Fr42169592&d=BQMCaQ&c=kbmfwr1Yojg42sGEpaQh5ofMHBeTl9EI2eaqQZhHbOU&r=C6Pt5AGtImanmAdcooarL-JZO8M5dSFPfs3VweYXYkE&m=uQ7CPzS3M1PvOW-L7Td3LKFDGNf9xsihVVMJIIX9taM&s=q1CMUCKocKU-gNbSU-b7-J4ET_q6ciQGvGNZKjFmaI8&e=>:

> @@ -131,7 +132,8 @@ sub read_whole_problem_file {
>
>  sub read_whole_file {
>       my $filePath = shift;
> -
> +     warn "<br/>Can't read file $filePath<br/>" unless -r $filePath;


Why is there the initial
here? It makes the error message look a little wierd.

-
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_openwebwork_pg_pull_245_files-23r42169592&d=BQMCaQ&c=kbmfwr1Yojg42sGEpaQh5ofMHBeTl9EI2eaqQZhHbOU&r=C6Pt5AGtImanmAdcooarL-JZO8M5dSFPfs3VweYXYkE&m=uQ7CPzS3M1PvOW-L7Td3LKFDGNf9xsihVVMJIIX9taM&s=dKDiHSkZYYvz1jMHrOY0vIVqD2f_9MpM9K3Ubw-n8RM&e=>.